examples/charm++/ckio/iotest.ci

```diff
@@ -0,0 +1,89 @@
+mainmodule iotest {
+	include "ckio.h"; // includes the necessary functions for CkIO
+	mainchare Main {
+		entry Main(CkArgMsg* m);
+		entry void ready(Ck::IO::FileReadyMsg* msg);
+		entry void startWrite(Ck::IO::SessionReadyMsg* msg);
+		entry void postWrite(CkReductionMsg* msg);
+		entry void close(CkReductionMsg* msg);
+		entry void decrementRemaining();
+
+		entry void startWritingCycle(int file_number){
+			serial {
+				/**
+				Set the Options struct so that CkIO opens the file and sets the correct configuration for writing.
+				The writeStripe parameter determines the amount of contiguous bytes each writer chare will write to file.
+				The peStripe parameters determines how much actual data each writer chare will aggregate at a given time. This is used so that 
+				a bunch of tiny data gets distributed across many writing chares when it would be better to all go to a single chare. 
+				It is required that peStripe >= writeStripe.
+				*/
+				ckout << "starting the writing cycle for " << file_number << endl;
+				Ck::IO::Options opts; // struct containing the options for the writer
+				opts.writeStripe = 1024; // collect up to 1kB of data before writing; use the specific number you'd like or it defaults to 4MB
```

Comment:
I know this PR is old and probably just collecting dust on a shelf, but in terms of exemplifying CkIO usage, the values should probably be in the range of megabytes, to be comparable in scale to typical Lustre stripe sizes. The original design was that they should be equal to (a multiple of) the FS stripe size, to limit the contention on storage servers and the individual stripes.